### DISCUSSION TITLE
Multiple congregations that share the same Windows Desktop?

### DISCUSSION THREAD

**Discussion by luisenriquereyes** [sircharlo/meeting-media-manager#237] [Q&A] [answered]
Greetings from the Dominican Republic.  Thank you so very much for this very useful applications. It saves our congregation so much time.  Now back to in person meetings, our Kingdom Hall shares the same Windows desktop with three congregations.  Will there be any problems with the Meeting date and time setup configuration?  It seems to only allow for one Midweek meeting and one Weekend meeting schedule.   Our three congregations have different dates and times.

**Comment by sircharlo** [comment, marked as answer]
Awesome idea. Keep an eye out for the next release! 😃

**Reply by sircharlo** [reply]
Implemented multiple congregation support in https://github.com/sircharlo/jw-meeting-media-fetcher/commit/3a1c4317a6ba629f7c27a2ed5df9db641d7c5d9e

**Reply by luisenriquereyes** [reply]
Hello my brother.  I see your message that says "Implemented multiple congregation support"   I guess we wait until the next release for it to become available?

**Reply by sircharlo** [reply]
That's correct; should be out later today if all goes well!
